config/introspection.go

```diff
@@ -50,6 +52,17 @@ type Jaeger struct {
 	BufferMax   int    `yaml:"buffer_max,omitempty" json:"buffer_max,omitempty"`
 }
 
+// Sentry is the [Sentry] specific tracing configuration.
+//
+// [Sentry]: https://sentry.io
+type Sentry struct {
+	// DSN to be passed to [github.com/getsentry/sentry-go.ClientOptions.Dsn].
+	DSN string `yaml:"dsn" json:"dsn"`
+	// Environment to be passed to
+	// [github.com/getsentry/sentry-go.ClientOptions.Environment].
+	Environment string `yaml:"environment,omitempty" json:"environment,omitempty"`
```

Comment:
Presumably this will be `stage`, `prod` etc?

Reply:
Regarding `Debug`, do we want to allow users to toggle that or could we eventually key it off of `clair.config.log_level`.

Reply:
I don't think anything else makes sense, `dist` I'd imagine is something we want to describe the Clair version (and all it's facets).

Reply:
I think the `Debug` spews stuff out to a configured writer, so we'd have to cook something up to repackage that into our logs. I think I can/will add this if it ends up being needed.

Reply:
That is to say: yes, we should control that via the main logging tunable, but it'll complicate the integration a little bit.